code/modules/religion/religion_sects.dm

```diff
@@ -0,0 +1,155 @@
+/datum/religion_sect
```

Comment:
If you introduce a new datum system like this autodoc would be nice

Reply:
smh cobby you trash

Reply:
😏